services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImpl.java

```diff
@@ -166,12 +166,12 @@ public News updateNews(News news, String updater, Boolean post, boolean publish)
         if (news.getTargets() == null || news.getTargets().isEmpty()) {
           news.setTargets(oldTargets);
         }
-        publishNews(news, updater);
+        publish(news, updater);
```

Comment:
As discussed together last time, should we instead of that just restore the old code and nomore create symlink when publishing an article which will avoid [the last fixed problem](https://github.com/exoplatform/news/commit/f8c35f2fcc58bf37cb93ae0717d680acfb0fb763) 
@GouadriaHanen @ahamdi